internal/inventory/inventory.go

```diff
@@ -28,6 +28,7 @@ func (s Inventory) IsAccountOnInventory(name string) bool {
 // AddAccount adds a new account into the Inventory
 func (s *Inventory) AddAccount(account Account) error {
 	if s.IsAccountOnInventory(account.Name) {
+		// TODO Should not be error?
```

Comment:
I consider it an error since we can't add the same account to the inventory twice